src/tools/tidy/src/ext_tool_checks.rs

```diff
@@ -491,6 +514,25 @@ fn shellcheck_runner(args: &[&OsStr]) -> Result<(), Error> {
     if status.success() { Ok(()) } else { Err(Error::FailedCheck("shellcheck")) }
 }
 
+/// Check that spellchecker is installed then run it at the given path
+fn spellcheck_runner(args: &[&str]) -> Result<(), Error> {
+    match Command::new("typos").arg("--version").status() {
+        Ok(_) => (),
+        Err(e) if e.kind() == io::ErrorKind::NotFound => {
+            return Err(Error::MissingReq(
+                "typos",
+                "spellcheck file checks",
+                // sync version with .github/workflows/ci.yml
+                Some("install tool via `cargo install typos-cli@1.34.0`".to_owned()),
```

Comment:
Does it have to be v1.34.0? Would it make more sense to omit the version number? Seems like "the latest version" would be better.

Reply:
And s/ci.yml/spellcheck.yml/, as above.

Reply:
I've added commit with check that ensures that correct version is installed. In case if version differs, output from local check and CI will diverge (which is bad). This should probably be not fatal error in case if people have different version for whatever case.